### PR TITLE
Update CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by FranBonath**
The code of conduct was still in v1.0. I copied the current COC from the website

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

**Review by FriederikeHanssen — APPROVED** [review]

**Review by mirpedrol — APPROVED** [review]
Thanks!
Could you also update `CHANGELOG.md`?

File: CODE_OF_CONDUCT.md
Comment (mashehu) [inline CODE_OF_CONDUCT.md:1]: this is actually not the CoC that is downloaded with `nf-core create`, you have to look in `pipeline-template/` for that one

Comment (mirpedrol) [reply CODE_OF_CONDUCT.md:1]: Oh that's right! The pipelines CoC is [here](https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/CODE_OF_CONDUCT.md)

**Comment by FranBonath** [comment]
I need to redo this, sorry. I started the PR from changes to master. Closing this one and will open a new PR.

**Pull request closed without merging**